components/tooltip/index.css

```diff
@@ -15,19 +15,19 @@ governing permissions and limitations under the License.
 
 .spectrum-Tooltip {
 	--spectrum-tooltip-animation-duration: var(--spectrum-animation-duration-100);
+	--spectrum-tooltip-animation-distance: var(--spectrum-spacing-75);
 
 	/* override if additional spacing to source is required */
 	--spectrum-tooltip-margin: 0px;
 
 	--spectrum-tooltip-height: var(--spectrum-component-height-75);
 	--spectrum-tooltip-max-inline-size: var(--spectrum-tooltip-maximum-width);
-	--spectrum-tooltip-border-radius: var(--spectrum-corner-radius-100);
-
-	--spectrum-tooltip-icon-width: var(--spectrum-workflow-icon-size-50);
-	--spectrum-tooltip-icon-height: var(--spectrum-workflow-icon-size-50);
+	--spectrum-tooltip-border-radius: var(--spectrum-corner-radius-400);
 
 	--spectrum-tooltip-font-size: var(--spectrum-font-size-75);
```

Comment:
I think it would take care of _most_ of them, and honestly it's an interesting idea that I played around with for a bit after you mentioned it, but it doesn't cover the line-height issues--[line-height for the body resolves to line-height-200](https://github.com/adobe/spectrum-css/blob/main/tokens/dist/index.css#L1413C37-L1413C61), which is too big, and we need a custom one anyway. You're right that it's supposed to grow, if there's more than what fits on 1 line, if we have cjk line heights we'd want to accommodate those, etc., but I think at a minimum there needs to be a circumstance where it does fit into that 30px space similar to how the line-height needs to be adjusted to make button fit a minimum height.